### DISCUSSION TITLE
Can the "keenetic" packages be used on generic devices?

### DISCUSSION THREAD

**Discussion by roisiano** [Entware/Entware#905] [Q&A] [answered]
Hello:

My question is whether the "keenetic" packages, for example, astra-sm:
https://bin.entware.net/aarch64-k3.10/keenetic/astra-sm_0.2.88-4_aarch64-3.10.ipk

can be used in generic devices (in my case, aarch64; a S905X3 TV-Box, using CoreELEC) via entware, manually installing the .ipk.

Or should there be a specific .ipk here?
https://bin.entware.net/aarch64-k3.10/

Kind regards

**Comment by zyxmon** [comment, marked as answer]
We have astra in the main repo. This is a special version of astra for DVB on keenetic.
You can test if a special version works for you.

**Reply by roisiano** [reply]
Hello:

Thanks for the reply.

Are "astra" ( http://bin.entware.net/aarch64-k3.10/astra_4.4.188-2_aarch64-3.10.ipk ) and "astra-sm" (keenetic: http://bin.entware.net/aarch64-k3.10/keenetic/astra-sm_0.2.88-4_aarch64-3.10.ipk ) the same package? 

I have been "browsing" the .ipk files and they seem quite different.

In particular, the keenetic package has a "t2mi_decap" binary that is not present in "astra".

Kind regards
